pkg/ingress/compare.go

```diff
@@ -107,14 +107,14 @@ func (c *Controller) CompareResources(ctx context.Context) error {
 			// todo ApisixUpstream and ApisixPluginConfig
 			// ApisixUpstream and ApisixPluginConfig should be synced with ApisixRoute resource
 
-			// ApisixSSL
+			// ApisixSSL TODO: Support v2?
```

Comment:
Will v2 support be put in the next PR? Or is it done in this PR?

Reply:
Maybe in the next PR, this file also contains ApisixRoute needs to be changed. I'd like to change them together.